tests/functional/ui/test_create_sc_and_pool_ui.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import validate_num_of_pgs
+from ocs_ci.framework.testlib import ui, skipif_ocs_version
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    bugzilla,
+    green_squad,
+)
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import run_cmd
+
+
+logger = logging.getLogger(__name__)
+
+
+@ui
+@tier1
+@green_squad
+@bugzilla("2253013")
+@skipif_ocs_version("<4.16")
+@pytest.mark.parametrize(
+    argnames=["replica", "compression"],
+    argvalues=[
+        pytest.param(*[2, False], marks=pytest.mark.polarion_id("OCS-6255")),
+    ],
+)
+class TestScAndPoolUserInterface(object):
+    """
+    Test to create storageclass and cephblockpool via user interface and validate pg_num
+
+    """
+
+    def test_sc_and_pool_ui_and_validate_pg_num(
```

Comment:
We already have an existing UI test that covers the most of these steps in this test. Should we just add the pg_num validation to that existing test instead of creating a new one?

https://github.com/red-hat-storage/ocs-ci/blob/4bbd16255e426318b3f9530b79510d6c5c3e0fd6/tests/cross_functional/ui/test_create_pool_block_pool.py

Reply:
I added it as a new test, I dint want to add steps to the existing test as the fix is available from 4.16 whereas the previous test can run from 4.8

Reply:
4.8 is quite a while back and versions shouldn't be a problem since we have stable branches created. I still feel we should simply add the pg_num validation to the existing test rather than creating a new test case, as I don't see the need for it. This is my opinion, but please also get input from the Black squad as it is a UI test

Reply:
Done